plugins/modules/source_control/gitlab/gitlab_deploy_key.py

```diff
@@ -29,6 +29,14 @@
     description:
       - GitLab token for logging in.
     type: str
+  api_oauth_token:
+    description:
+      - GitLab OAuth token for logging in.
+    type: str
+  api_job_token:
+    description:
+      - GitLab CI job token for logging in.
+    type: str
```

Comment:
The Right Way(TM) to do that is creating a doc_fragment with this text and make the modules point to it. If not in this PR, then please raise an issue to address that.

Reply:
Will make this change